crates/services/compression/src/ports/block_source.rs

```diff
@@ -1,11 +1,41 @@
-/// Type alias for fuel_core_types::blockchain::block::Block
-pub type Block = fuel_core_types::blockchain::block::Block;
+use fuel_core_services::stream::BoxStream;
+
+/// Block with metadata needed for compression
+#[cfg_attr(any(test, feature = "test-helpers"), derive(Default))]
```

Comment:
If we ensure `test-helpers` is enabled as a dev-dependency like we do in other crates (for example `fuel-core-storage`, this can be simplified.
```suggestion
#[cfg_attr(feature = "test-helpers", derive(Default))]
```